es6-lib/consumer.js

```diff
@@ -0,0 +1,55 @@
+import Stomp from 'stomp-client';
+import Spatial from './services/spatial';
+import logger from './util/logger';
+import ISS from './upstream/iss';
+
+function consumer(config, zookeeper, metrics) {
+  const hosts = config.amq.host
+    .replace(/stomp:\/\//g, '')
+    .split(',')
+    .map(hostAndPort => hostAndPort.split(':'));
+
+  const reconnectOpts = {
+    retries: 3,
+    delay: 500
+  };
+
+  var hostIndex = 0;
+
+  const connect = () => {
+    const [host, port] = hosts[hostIndex % hosts.length];
```

Comment:
Are the GeoImports messages going onto both queues? Or are they put on either one randomly? You'll need to have a way to consume off both hosts, I think, rather than just using one as a failover.